addons/mail/static/src/components/chatter_container/chatter_container.js

```diff
@@ -55,27 +71,69 @@ class ChatterContainer extends Component {
     // Private
     //--------------------------------------------------------------------------
 
+    /**
+     * @private
+     */
+    _insertFromProps(props) {
+        const values = Object.assign({}, props);
+        if (values.threadId === undefined) {
+            values.threadId = clear();
+        }
+        if (!this.chatter) {
+            this.chatter = this.env.models['mail.chatter'].create(values);
+        } else {
+            this.chatter.update(values);
+        }
+    }
+
     /**
      * @private
      */
     _update() {
         if (this.chatter) {
-            const values = Object.assign({}, this.props);
-            if (values.threadId === undefined) {
-                values.threadId = clear();
-            }
-            this.chatter.update(values);
+            this.chatter.refresh();
```

Comment:
Looks like this will make lots of RPCs, whenever chatter container is re-render.

Reply:
Yes this is bad, but it's the best we can do. The worse part is not calling refresh often actually, but it's the fact refresh itself is making plenty of rpc instead of just one.

Reply:
Need to make a pass after this PR to at least make them all shadow, because it's currently not the case.